### PR TITLE
Create Post for This Month in Open Source Design

### PR DESCRIPTION
**Pull request by simonv3**
# What's Changed

* Added rake job for adding a post `rake post title="Post Title" [date=2015-02-02] [cat="News"]`
* Referenced the site image directly. Dunno if this is necessary, but it was part of an attempt to make all the jekyll blogs work nice together.
* Added a This Month in Open Source blog post. 

If you want to make edits to the text, do it here please: http://piratepad.net/opensourcedesign-monthly
(and then leave me a note on here or IRC so that I know to look)

### CODE REVIEW

**Comment by bnvk** [comment]
@simonv3 thanks so much for doing this excellent roundup?

File: _posts/2015-05-18-this-month-in-open-source-design.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+* We launched [the site](http://opensourcedesign.net/) [[irc](https://botbot.me/freenode/opensourcedesign/msg/37529727/)]!
+* Job postings are [now live](http://opensourcedesign.net/jobs)!
+* Events are [now live](http://opensourcedesign.net/events)!
+* Some of our members have started work on [Slick](https://github.com/opensourcedesign/slick), a theme for the Shout web app.
+
+# Events
+
+* We now have an [events page](http://opensourcedesign.net/events/)
+* We attended and presented at OpenTechSummit in Berlin [[irc](https://botbot.me/freenode/opensourcedesign/msg/37270321/)]. Jan Dittri presented on the difficulties present when designing for open source projects. We also had a nice little meetup at OTS with some   contributors on the project.
+* We were at the Libre Graphics Meeting in Toronto [[irc](http://libregraphicsmeeting.org/)].  Jan-Christoph Borchardt presented on Open Source Design and his experiences working on ownCloud. There was also a small workshop on design-reviews of each other’s projects.
```
Comment (jancborchardt) [inline _posts/2015-05-18-this-month-in-open-source-design.markdown]: Why does it say [irc] here? I guess a typo, same as below for the next link?

**Comment by jancborchardt** [comment]
:+1: great stuff @simonv3! Just the `[irc]` link label possible typo I noted inline.

File: _posts/2015-05-18-this-month-in-open-source-design.markdown (hunk 1 above)
Comment (simonv3) [inline _posts/2015-05-18-this-month-in-open-source-design.markdown]: Yep, that's a copy paste error (using [^link] threw all sorts of errors).

**Comment by simonv3** [comment]
@bnvk Thanks!

**Comment by jancborchardt** [comment]
Tweeted at https://twitter.com/opensrcdesign/status/600379184997912576 – retweet it! :)

**Pull request merged**